.github/workflows/build.yml

```diff
@@ -18,20 +22,23 @@ jobs:
         with:
           fetch-depth: 0  # fetches all commits/tags
 
-      - name: Set dynamic variables
-        # this might seem odd but is the recommended way to set variables
-        # reference: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
-        run: |
-          pip install poetry
-          BUILD_VERSION=$(poetry version -s)
-          echo "BUILD_VERSION=$BUILD_VERSION+$GITHUB_SHA" >> $GITHUB_ENV
-          echo "IMAGE_NAME_WITH_TAG=quipucords:$BUILD_VERSION" >> $GITHUB_ENV
```

Comment:
Just for the record, there were multiple problems with this:

1. It doesn't work as intended. Idea was to create something like "1.2.3+sha" as git tag. But this new variable was only visible since next step, and instead images were tagged with "1.2.3"
2. Container tags can't use "+" sign
3. Expiration time is assigned to container image, not tag. If we create tags with git hash when building from main, then we either apply expiration to both tags (which is not ideal, because we want main to always resolve), or neither tag (which is not ideal, because now we have "1.2.3+sha" tag that never expires and takes space in quay).